Go/fibonacci/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/enarx/codex/Go/fibonacci
+
+go 1.14
```

Comment:
That's quite ancient, why not 1.17 at least? I think that was the most recent one supported by tinygo

Reply:
I just picked something old since no special features are used.